plugins/org.eclipse.n4js.transpiler/src/org/eclipse/n4js/transpiler/print/PrettyPrinterSwitch.java

```diff
@@ -1399,6 +1420,10 @@ private String quote(String txt) {
 		return '\'' + ValueConverterUtils.convertToEscapedString(txt, false) + '\'';
 	}
 
+	private String prependHashbang(String txt) {
+		return "#!" + txt;
+	}
```

Comment:
I wonder why we do not use the ValueConverter (N4JSStringValueConverter for strings, HashbangValueConverter for this, and others) to revert elements to source code.

Reply:
Good point. I'm not sure if there was a reason for this. But at least we are already reusing a utility method from the value converters in `PrettyPrinterSwitch#quote()`.

I also just noticed that there are several `#getValueAsString()` methods defined in `N4JS.xcore` (i.e. a third way of getting back to source code) and they do not seem to be used in all cases by `PrettyPrinterSwitch`.